### DISCUSSION TITLE
[Project beta] Hard to go back to repo

### DISCUSSION THREAD

**Discussion by jlei523** [community/community#11654] [Projects and Issues]
Once I go into a Project view, it's hard to navigate back to the repo that linked me to the beta project.

There is no link to go back to the repo. Also, the back button does not work because the back history gets polluted with project actions.

**Comment by dag7dev** [comment]
Kudos, I am unable to link a project to a repo (or simply I don't know how).

**Comment by Spartan2909** [comment]
I usually add a link to the repo to the project’s README, although I agree that it would be nice to have a link back to the repo built in
